### ISSUE TITLE
source_id registration of RegIPSL

### ISSUE THREAD

**Issue by LluisFB** [WCRP-CORDEX/cordex-cmip6-cv#122]
### label

RegIPSL

### label_extended

regional Earth system model of IPSL 

### source_id

RegIPSL

### source_type

AORCM

### release_year

2019

### activity_participation

DD

### institution_id

UBA-CIMA-IFAECI

### further_info_url

https://gitlab.in2p3.fr/ipsl/lmd/intro/regipsl/regipsl/-/wikis/home

### What license do you choose?

CC BY 4.0

**Comment by jesusff** [comment]
Looks good, thanks

This is a different version as compared to that to be used in Med-CORDEX, right? Please, check, the [simulation status for this model](https://wcrp-cordex.github.io/simulation-status/CMIP6_downscaling_plans.html?search.search=regipsl) and update the estimated completion dates (the current one is overdue) or status by opening an issue [here](https://github.com/WCRP-CORDEX/simulation-status/issues/new?title=Update%20status%20of%20RegIPSL).

**Comment by gnikulin** [comment]
It would be also good to discuss the registration of RegIPSL with IPSL to have consistent RCM acronym (versions etc.) for different CORDEX domains.

**Comment by LluisFB** [comment]
If everything goes fine, next year I should start to direct a PhD thesis that will mainly work in producing CORDEX CMIP7 over SA at 12.5 km. The model has quite evolve since I registered it. But, because no data has been published, I would say this is not a problem.

I am not sure if RegIPSL-EUR, would be the same as RegIPSL-SA. In this case, we should register another model, isn't it?

**Comment by jesusff** [comment]
Model versions will likely be needed. Please, do not register the model source_ids including the domain names. The domain is already part of the DRS and, in principle, nothing prevents using RegIPSL-EUR in any other region.

Do you have some news from Jan's version? Their simulations are also stale as planned since 2023. There is no official versioning of the RegIPSL?

**Comment by LluisFB** [comment]
Of course, model name does not have to include the domain. Was a way to explain it.

No, there is not an actual versioning of the RegIPSL. The code is in a repository, but we do not tag versions. 

Respect the 2023 Jan's simulation, if it is the one promised in the SA domain we will scrap it. The resolution was at 20 km and we suffered major setbacks from the GCM forcing (we were killing the Amazon forest in 1 year).

So, I am going to start from scratch, using CMIP7 data and 12.5 k SA domain. However, I can ensure that we are not going to be able to produce more than 20 years of data per simulation (Evaluation, Historical and Future). SA domain is so big that already 20 years is about 350 TB per simulation. Currently I do not have disk space to keep them. I have 2 years (we would start to simulate by 2028) to see how can I solve my space issues.

**Comment by sam-somot** [comment]
@LluisFB : IPSL has declared a RegIPSLv2 version in Med-CORDEX (not officially register on github yet). see [here](https://wcrp-cordex.github.io/simulation-status/CORDEX_CMIP6_status.html#MED-12). So good to coordinate with them (Jan, Romain) on the model naming. At CNRM, we use a letter to identify versions adapted to a given domain: CNRM-ALADIN64E1 for Europe, CNRM-ALADIN64C1 for Central America for example. In our case, we indicate the domain as we are re-tuning depending on the domain.

**Comment by LluisFB** [comment]
I see @sam-somot thank you.

I think somehow we are going to do the same. I am pretty sure that Med-CORDEX had different issues than SA-CORDEX (we are not able to activate the ocean, where to put the West limit in the Pacific?) .

This is something that from the Model Documentation side (which I am co-leading) will be necessary to make it clear. Since with this complex models there are a lot of different components to be activated and set-up.
